spring-reactive-modules/spring-5-reactive-client-2/src/main/java/com/baeldung/limitrequests/server/Concurrency.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.limitrequests.server;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.IntSupplier;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Concurrency {
```

Comment:
This class looks like a utility class so to prevent the instantiation let's add a private constructor.